docs/docs/configuration/index.md

```diff
@@ -39,6 +39,8 @@ It is not recommended to copy this full configuration file. Only specify values
 
 ```yaml
 mqtt:
+  # Optional: Enable mqtt server (default: shown below)
+  enabled: True
```

Comment:
> And if I set it to disabled by default then that would break every existing users setup by default. even if we had a huge font note in the changelog, there'd still be plenty of users who made issues because they didn't read the changelog and their automations stopped working.

I thought breaking changes were not much of a problem as long as properly documented, they seem to happen in almost every major 0.x releases anyway.

But it's up to you. It was just a suggestion :)

Reply:
You're definitely not wrong but trying to move in the direction of less breaking changes.